lib/server.js

```diff
@@ -153,6 +153,7 @@ Server.prototype.start = function () {
     password: this.password,
     passwordAdmin: this.admin_password,
     path: this.config.path,
+    logsPath: this.config.logsPath,
```

Comment:
Not a supported argument, https://github.com/Dahlgren/node-arma-server/blob/5685ffea362cce4c402d0dfd43c01808937823e6/src/server.js#L21-L63